boxoffice/templates/boxoffice.js

```diff
@@ -254,7 +254,7 @@ $(function() {
                 }
               });
 
-              if(valid_discount_coupon) {
+              if (valid_discount_coupon) {
                 boxoffice.ractive.set('order.line_items',line_items);
```

Comment:
Incase discount policy of the coupon is applicable to multiple items, wouldn't it be better to set `order.line_items` than setting each line_item.